torch_frame/_stype.py

```diff
@@ -62,11 +62,25 @@ def use_dict_multi_nested_tensor(self) -> bool:
 
     @property
     def use_multi_tensor(self) -> bool:
-        r"""This property indicates if the data of an stype is stored in
+        r"""This property indicates if the data of an
+        :class:`~torch_frame.stype` is stored in
         :class:`torch_frame.data._MultiTensor`.
         """
         return self.use_multi_nested_tensor or self.use_multi_embedding_tensor
 
+    @property
+    def parent(self):
+        r"""This property indicates if an :class:`~torch_frame.stype` is
+        user-facing or internal. Internal :class:`~torch_frame.stype` will
+        be mapped to its parent during materialization. For
+        :class:`<stypes>~torch_frame.stype` that are both internal and
+        user-facing, the parent maps to itself.
```

Comment:
Make it clear that user-facing is for specifying column type in `DataFrame`, while internal is for stype grouping in `TensorFrame`.